packages/wrangler/src/index.tsx

```diff
@@ -2016,9 +2016,7 @@ export async function main(argv: string[]): Promise<void> {
             const accountId = await requireAuth(config);
 
             await fetchResult<{ id: string }>(
-              `/accounts/${accountId}/storage/kv/namespaces/${encodeURIComponent(
```

Comment:
Trying to grok this change - so does `encodeURIComponent` not help here?

Reply:
Correct. Unless cloudflare decides to change the way it generates IDs, which is unlikely. I'm simply reverting this specific change that was put in the previous PR. I'm resisting adding this encoding here, because if so, we should probably add it for all URL interpolations; we probably _should_ do that, but that's a bigger PR and we should do that all at once. But it's not necessary now, and I don't want it's presence to confuse anyone who reads this and wonders why it's here and not everywhere else.

Reply:
The idea is that `namespace-id` is a UID which is expected never to have any characters that need encoding.